e2e/cypress/integration/enterprise/ldap/ldap_group_sync_spec.js

```diff
@@ -0,0 +1,343 @@
+
+    describe('LDAP Group Sync Automated Tests', () => {
+        beforeEach(() => {
+            // * Check if server has license for LDAP
+            cy.apiRequireLicenseForFeature('LDAP');
+
+            // # Initial api setup
+            cy.apiInitSetup().then(({team, user}) => {
+                testTeam = team;
+                testUser = user;
+
+                // # Update LDAP settings
+                cy.apiGetConfig().then(({config}) => {
+                    setLDAPTestSettings(config);
+                });
+
+                // # Login as sysadmin and add board-one to test team
+                cy.apiAdminLogin();
+
+                // # Link board group
+                cy.visit('/admin_console/user_management/groups');
+                cy.get('#board_group').then((el) => {
+                    if (!el.text().includes('Edit')) {
+                        // # Link the Group if its not linked before
+                        if (el.find('.icon.fa-unlink').length > 0) {
+                            el.find('.icon.fa-unlink').click();
+                        }
+                    }
+                });
+
+                // # Link developers group
+                cy.visit('/admin_console/user_management/groups');
+                cy.get('#developers_group').then((el) => {
+                    if (!el.text().includes('Edit')) {
+                        // # Link the Group if its not linked before
+                        if (el.find('.icon.fa-unlink').length > 0) {
+                            el.find('.icon.fa-unlink').click();
+                        }
+                    }
+                });
+
+                // # Create a test channel
+                cy.apiCreateChannel(testTeam.id, 'ldap-group-sync-automated-tests', 'ldap-group-sync-automated-tests').then((response) => {
+                    testChannel = response.body;
+                });
+            });
+        });
+
+        it('MM-T1537 - Sync Group Removal from Channel Configuration Page', () => {
+            // # Login as sysadmin and link 2 groups to testChannel
+            cy.apiAdminLogin();
+            cy.visit(`/admin_console/user_management/channels/${testChannel.id}`);
+            cy.wait(2000); //eslint-disable-line cypress/no-unnecessary-waiting
+
+            // # Link first group
+            cy.findByTestId('addGroupsToChannelToggle').click();
+            cy.get('#multiSelectList').should('be.visible');
+            cy.get('#multiSelectList>div').children().eq(0).click();
+            cy.get('#saveItems').click();
+
+            // # Link second group
+            cy.findByTestId('addGroupsToChannelToggle').click();
+            cy.get('#multiSelectList').should('be.visible');
+            cy.get('#multiSelectList>div').children().eq(0).click();
+            cy.get('#saveItems').click();
+
+            // # Click save settings on bottom screen to save settings
+            cy.get('#saveSetting').should('be.enabled').click({force: true});
```

Comment:
Is `{force: true}` necessary for this click? Same goes for the other ones.

Reply:
Guess not :P I guess it's because previously it was needed (it wouldn't work without it) and I just got into the habit of using it for saveSetting. However, I just tried without the force and everything worked. Removed now. Thanks!